### PR TITLE
Fix missing env

### PR DESCRIPTION
**Pull request by yeelauren**
Hey friends! 
While testing locally, I noticed that the `environment.yml` file seems to be missing some packages for the notebooks in `/docs`. 
I've made my attempt at fixing the version numbers. 

I also noticed that this repo is missing a reference to the submodule [stacchip ](https://github.com/Clay-foundation/stacchip)

https://github.com/Clay-foundation/model/blob/a61f9faf4cadfdbef76b0651ae0aa638ef396323/docs/clay-v1-wall-to-wall.ipynb#L56

To get this to run locally, I had to copy and move the directory around for stacchip.

### CODE REVIEW

**Comment by yellowcap** [comment]
Not sure if we should also update the `conda-lock.yml` file with this?

**Review by weiji14 — COMMENTED** [review]
Hi @yeelauren! Thanks for opening this, the conda environment definitely needs an update with all the recent changes. Could you refresh the `conda-lock.yml` file as @yellowcap mentioned after adding `stacchip` (see suggestion below)? You can do this by following the instructions at https://clay-foundation.github.io/model/installation.html#advanced, i.e. run:

```bash
conda-lock lock --mamba --file environment.yml --with-cuda=12.0
```

File: environment.yml
[the remark this answers is not in the document]
Comment (yellowcap) [reply environment.yml:1]: stacchip is pip installable now 😄  https://pypi.org/project/stacchip/ So the addition should read

```
  - pip
  - pip:
     - stacchip==0.1.30
```

**Comment by yellowcap** [comment]
We removed the stacchip library from the dependencies, as it was only used in one notebook for one tiny function. The updated wall-to-wall notebook is now independent of stacchip.

I think with this also the geoarrow and boto3 dependencues are no longer necessary. @yeelauren when you have time, please test your workflow again using main or the branch and let us know if the dependency issues were resolved.

Btw, we are also updating other libraries on the env file in https://github.com/Clay-foundation/model/pull/264

**Comment by yellowcap** [comment]
I think this has been fixed in updating the env here https://github.com/Clay-foundation/model/pull/264 and removing stacchip from the wall-to-wall example here https://github.com/Clay-foundation/model/pull/263.

@yeelauren feel free to reopen if you still encounter these problems.

**Pull request closed without merging**